starting_from_scratch_lab/lab 37/lab_37_tar3.js

```diff
@@ -0,0 +1,50 @@
+let game = [
+  ["X", "O", "O"],
+  ["O", "X", "O"],
+  ["O", "O", "X"],
+]; // diag
+
+let game = [
+    ["X", "O", "O"],
+    ["X", "O", "O"],
+    ["O", "O", "X"],
+  ]; // colm
+
+  let game = [
+    ["X", "X", "X"],
+    ["O", "X", "O"],
+    ["O", "O", "X"],
+  ]; // row
+
+const verdict = theWinnerTakesItAll(game);
+
+function theWinnerTakesItAll(game) {
```

Comment:
I would define a helper function for example:

```
function allEqual(...values) {
  return values => values.every( v => v === arr[0] )
}
```

And then used it in the code for example:

```
if (allEqual(game[0][0], game[0][1], game[0][2]) {
  return game[0];
}

// and so on in a loop
```